nemo/collections/asr/parts/submodules/rnnt_beam_decoding.py

```diff
@@ -325,6 +328,9 @@ def __init__(
         else:
             self.ngram_lm = None
 
+        self.hat_subtract_ilm = hat_subtract_ilm
+        self.hat_ilm_weight = hat_ilm_weight
```

Comment:
If this is set, assert that hasattr(self.joint, "return_hat_ilm"). Only assert, don't set it yet